src/sync/models/decentralizedOracle.js

```diff
@@ -40,8 +40,8 @@ class DecentralizedOracle {
       amounts: _.fill(Array(this.numOfResults), '0'),
       resultIdx: null,
       blockNum: this.blockNum,
-      startBlock: this.blockNum,
-      endBlock: this.arbitrationEndBlock,
+      startTime: this.blockNum,
```

Comment:
not this.blockNum

Reply:
changed it to 0 as the default. makes more sense than using the blockNum since its timestamp vs number.